t2kdm/backends.py

```diff
@@ -1081,7 +1081,7 @@ def _remove(self, surl, lurl, last=False, verbose=False, **kwargs):
         if not ret["OK"]:
             raise BackendException("Failed: %s" % (ret["Message"]))
 
-        for lurl, error in ret["Value"]["Failed"].items():
+        for lurl, error in list(ret["Value"]["Failed"].items()):
```

Comment:
This `list` is not needed, since we iterate directly over the output of `items`.